dallinger/command_line/ec2.py

```diff
@@ -100,17 +100,15 @@ def list__instance_types(ctx, region):
 @click.option("--region", default=None, help="Region name")
 @click.option("--type", default="m5.xlarge", help="Instance type")
 @click.option("--storage", default=32, type=int, help="Storage in GB; default is 32 GB")
-@click.option(
-    "--pem", default="dallinger", help="PEM file name; default is dallinger.pem"
-)
+@click.option("--pem", default=None, help="PEM file name; default is dallinger.pem")
 @click.option(
     "--image_name",
     default="ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-20230516",
     help="Image name; default is Ubuntu 22.04",
 )
 @click.option(
     "--security_group_name",
-    default="dallinger",
+    default=None,
     help="Security group name; default is dallinger",
```

Comment:
Should mention that by default this comes from config.txt

Reply:
Updated the instructions now